README.md

```diff
@@ -221,8 +221,8 @@ Check the list of pipelines you have created using the CLI:
 ```
 $ tkn pipeline ls
 
-NAME                       AGE              LAST RUN   STARTED   DURATION   STATUS
-petclinic-deploy-pipeline  25 seconds ago   ---        ---       ---        ---
+NAME     AGE              LAST RUN   STARTED   DURATION   STATUS
+voting   25 seconds ago   ---        ---       ---        ---
```

Comment:
Dev console shows an incorrect graph for this pipeline. Could you verify? Shows `build-ui` last

<img width="640" alt="Screenshot 2019-11-20 at 08 41 42" src="https://user-images.githubusercontent.com/1174934/69243744-ad9a4500-0b71-11ea-84b8-0ffceea14832.png">